superset/assets/src/dashboard/components/Dashboard.jsx

```diff
@@ -86,9 +86,7 @@ class Dashboard extends React.PureComponent {
 
   componentWillReceiveProps(nextProps) {
     if (!nextProps.dashboardState.editMode) {
-      const version = nextProps.dashboardState.isV2Preview
-        ? 'v2-preview'
-        : 'v2';
+      const version = 'v2';
```

Comment:
nit I wouldn't define a variable for this, you can just define it as part of the object below.

Reply:
fixed.